debian/changelog

```diff
@@ -1,3 +1,49 @@
+wiringpi (3.1-1) unstable; urgency=low
```

Comment:
we do not consider published releases to be unstable :wink:

Reply:
also, the changelog will most likely need to be autogenerated going forward.

Reply:
Changed to stable.